lib/jquery-rjs/rendering.rb

```diff
@@ -1,13 +1,21 @@
 require 'action_view/helpers/rendering_helper'
 
-ActionView::Helpers::RenderingHelper.module_eval do
-  def render_with_update(options = {}, locals = {}, &block)
-    if options == :update
-      update_page(&block)
-    else
-      render_without_update(options, locals, &block)
-    end
+module JqueryRjs::RenderingHelper
+  method_name = "render#{'_with_update' if RUBY_VERSION < '2'}"
+  define_method(method_name) do |options = {}, locals = {}, &block|
+    update_page(&block) if options == :update
+
+    args = options, locals, block
+    RUBY_VERSION < '2' ? render_without_update(*args) : super(*args)
```

Comment:
So... you'd rather see different methods entirely with one RUBY_VERSION check than a meta'd single method with multiple version checks?

`if RUBY_VERSION < 2 alias_method_chain with `render_with_update`, else prepend render, a separately defined method` ?